AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -13,6 +15,8 @@
 
 @implementation MSChannelGroupDefault
 
+static volatile uint32_t _hasEnteredApplicationWillTerminate;
```

Comment:
A more direct form may be more readable?
```suggestion
static volatile uint32_t _applicationWillTerminate;
```

Reply:
Will be confusing now that it is a property